docs/cloud-native-security/vuln-management-findings.asciidoc

```diff
@@ -1,32 +1,54 @@
 
-image::images/cnvm-findings-page.png[The Vulnerabilities tab of the Findings page]
+Clicking on a finding provides a detailed description of the vulnerability, and any available remediation information.
 
 
 [discrete]
 [[vuln-findings-grouping]]
-== Group, sort, and filter findings
+== Group and filter findings
+
+To help you prioritize remediation efforts, you can organize findings in various ways.
+
+[discrete]
+=== Group findings
+
+. Click **Group vulnerabilities by:** to open a list of fields by which you can group your data. Select one of the suggested fields, or click **Custom field** to choose your own. You can select up to three group-by fields at once. 
+. When grouping is turned on, click a group to expand it and examine all sub-groups or findings within that group.
+. To turn off grouping, click **Group findings by:** and select **None**.
 
-You can group your data by resource by selecting *Resource* from the *Group by* menu. When data is grouped by resource, you can click on the name of a virtual machine to view all vulnerabilities that were found on it.
+NOTE: Multiple groupings apply to your data in the order you selected them. For example, if you first select **Cloud account**, and then select **Resource**, the top-level grouping will be based on **Cloud account**, and its subordinate grouping will be based on **Resource**, as demonstrated in the following screenshot:
 
 image::images/cnvm-findings-grouped.png[The Vulnerabilities tab of the Findings page]
 
+[discrete]
+[[cnvm-filter-findings]]
+=== Filter findings
+You can filter the data in two ways:
 
-When *Group by* is set to *None*, you can sort the Findings table by clicking the column headings or the *Sort fields* button to the upper left of the table. When sorting is active, the *Sort fields* button changes to *X fields sorted* (where _X_ is the number of fields sorting your data), and can be used to modify or clear sorting.
+* *The KQL search bar*: For example, search for `vulnerability.severity : "HIGH"` to view high severity vulnerabilities.
+* *In-table value filters*: Hover over a finding to display available inline actions. Use the **Filter In** (plus) and **Filter Out** (minus) buttons.
 
-Independent of grouping, you can filter data in two ways:
+[discrete]
+[[cnvm-customize-the-findings-table]]
+=== Customize the Findings table
+When grouping is turned off, you can use the toolbar buttons in the upper-left of the Findings table to select which columns appear:
 
-- *The KQL search bar*: Use this to filter your findings. For example, search for `vulnerability.id : CVE-2019-00001` to view all findings related to a particular vulnerability.
+* **Columns**: Select the left-to-right order in which columns appear.
+* **Sort fields** or **_n_ fields sorted**: Sort the table by one or more columns, or turn sorting off.
```

Comment:
```suggestion
* **Sort fields**: Sort the table by one or more columns, or turn sorting off.
```
Looking at the updated screenshots, it looks like this copy has changed and it says **Sort fields _n_** even when sorting is on.